### ISSUE TITLE
Schedule the sending of DMs to a certain hour and timezeone by the minute

### ISSUE THREAD

**Issue by mb20231** [Oxlac/MR.DM#14]
Schedule the sending of DMs to a certain hour and timezeone by the minute

**Comment by clut19** [comment]
Does this work for you? mine opens the new google chrome tab but doesn't send any dm.

**Comment by mb20231** [comment]
No, I wasn't even able to install I'm not a dev or savvy, I use other plugins available out there on chrome but they work differently not 100% manual
